### PR TITLE
Feedback route of superuser

### PR DESCRIPTION
**Pull request by greykoalacode**
## Description of the change

> Added feedback route in superuser.. Added inline replies with it. Do check and let me know

## Type Of Change
- [x] Bug Fix (non-breaking change that fixes an issue)
- [*] New Feature (non-breaking change that adds functionality)
- [x] Breaking Change (fix or feature that would cause existing functionality to not work as expected)

## Related Issues

> Fix [#<ISSUE_NUMBER>](<ISSUE_LINK>) 

## Development & Code Review 
- [*] Fixed all the Eslint errors (run `npm run lint-fix`)
- [*] All ESLint rules passes locally (run `npm run lint`)
- [*] The code changed/added has been covered with tests
- [*] All the test cases added/changed are passing

### CODE REVIEW

File: src/Components/superuser/feedback/FeedbackCard/FeedbackCard.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        <img
+          className="mr3"
+          alt="user-icon"
+          style={{ borderRadius: '50%' }}
+          height="35vh"
+          width="auto"
+          src="https://res.cloudinary.com/siesgstarena/image/upload/f_auto,q_auto/v1546283328/arena/assets_webp/gravatar.webp"
+        />
+        {
+            (isReply || isReplied) ? (
```
Comment (ac030540) [inline src/Components/superuser/feedback/FeedbackCard/FeedbackCard.jsx:53]: Why are you using isReply and isReplied together? Technically they are pointing to the same value, right?

Comment (greykoalacode) [reply src/Components/superuser/feedback/FeedbackCard/FeedbackCard.jsx:53]: > You forgot to add Heading on the page plus you didn't add the details about the feedback (time and date).

Yes  i had forgot to add them , will replicate them

**Comment by greykoalacode** [comment]
> @greykoalacode The code looks very professional good going :-)

Thank you @Swap76 !

File: src/Components/superuser/feedback/index.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <div className="mw7 pa2 center">
+      <Headline5 style={{margin: '0.2em'}}>Feedbacks</Headline5>
+      {feedbacks.map(
+        (feedback, index) => {
+          const {
+            name, email, message,
+            isReplied, replyVal, responseTime, responseDate,
+          } = feedback;
+          return (
+            <div key={index.toString()}>
```
Comment (ac030540) [inline src/Components/superuser/feedback/index.jsx:19]: Never use index as key. Not even as string.

File: src/Components/superuser/feedback/index.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import feedbacks from './feedbacks';
+import FeedbackCard from './FeedbackCard/FeedbackCard';
+import CustomSnackbar from '../../common/Snackbar/index';
+import { Headline5 } from '@material/react-typography';
+
+const FeedbackContainer = () => {
+  const [snackbarMessage, setSnackbarMessage] = useState('');
+  return (
+    <div className="mw7 pa2 center">
+      <Headline5 style={{margin: '0.2em'}}>Feedbacks</Headline5>
```
Comment (ac030540) [inline src/Components/superuser/feedback/index.jsx:11]: Make the style of Headline similar to the rest of the routes. Also add a short description about the page with Body2. You can find that  description from arena repo.

File: src/Components/superuser/feedback/FeedbackCard/FeedbackCard.jsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  const addReply = () => {
+    setReplyStatus(true);
+    setSnackbarMessage('Replied Successfully');
+  };
+
+  return (
+    <div className="b--light-gray ba br3 mb3 shadow-4">
+      <div className="flex justify-between items-center mb2">
+        <div className="flex flex-column mt2">
+          <Headline6 style={{ margin: '0em 0em 0em 0.6em', lineHeight: '1.5rem' }}>{user}</Headline6>
```
Comment (ac030540) [inline src/Components/superuser/feedback/FeedbackCard/FeedbackCard.jsx:27]: To specify margins you can use tachyons.

File: src/Components/superuser/feedback/FeedbackCard/FeedbackCard.jsx [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+          <small style={{ margin: '0.1em 0em 0.1em 1em' }} className="pointer">{email}</small>
+          <small style={{ margin: '0.1em 1em' }} className="pointer">{`${responseTime}, ${responseDate}`}</small>
```
Comment (ac030540) [inline src/Components/superuser/feedback/FeedbackCard/FeedbackCard.jsx:29]: Make them mid-gray colored.

File: src/Components/superuser/feedback/FeedbackCard/FeedbackCard.jsx [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+  const toggleOpen = () => setOpen(!isOpen);
+  const disableReply = (isReply);
+  const replyStatus = (isReply) ? 'https://img.icons8.com/material-rounded/24/6200ee/double-tick.png' : 'https://img.icons8.com/ios-glyphs/24/6200ee/paper-plane.png';
+  const addReply = () => {
+    setReplyStatus(true);
+    setSnackbarMessage('Replied Successfully');
+  };
+
+  return (
+    <div className="b--light-gray ba br3 mb3 shadow-4">
```
Comment (ac030540) [inline src/Components/superuser/feedback/FeedbackCard/FeedbackCard.jsx:24]: Remove shadow, it looks a lot cleaner without it.

File: src/Components/common/CustomBubble/CustomBubble.jsx [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Body2 } from '@material/react-typography';
+import PropTypes from 'prop-types';
+import './CustomBubble.css';
+
+const CustomBubble = ({ content, className, bubbleType }) => {
+  const chatClass = (bubbleType === 'chat') ? 'tri-right right-top' : '';
+  return (
+    <div className={`talk-bubble ${chatClass} ${className}`}>
+      <Body2 className="" style={{ margin: '0em' }}>{content}</Body2>
```
Comment (ac030540) [inline src/Components/common/CustomBubble/CustomBubble.jsx:10]: Try classNames= "black-60" or "black-70".

File: src/Components/superuser/feedback/FeedbackCard/FeedbackCard.jsx (hunk 5 above)
Comment (ac030540) [reply src/Components/superuser/feedback/FeedbackCard/FeedbackCard.jsx:29]: use Body2 instead of small

**Comment by greykoalacode** [comment]
Thank you @ac030540

**Pull request merged**